README.md

```diff
@@ -16,3 +16,9 @@
 Confirm group based consensus on messages in a p2p network
 
 #Todo Items
+
```

Comment:
Missing version heading for this change set. Would be good to specify the version(guessing something like `### [0.1.1] - RUST-3 Tasks`)